internal/config/env.go

```diff
@@ -57,9 +58,12 @@ func FileName() string {
 }
 
 func LoadEnv() error {
+	const (
+		localEnvFile = "local"
```

Comment:
should this be named "localEnvName"?

Reply:
Yeah seems more appropriate, I will change it

Reply:
I feel the more appropriate name would be `localEnvName`